core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BpmBillingProjectLifecycle.scala

```diff
@@ -75,11 +76,17 @@ class BpmBillingProjectLifecycle(
                                  config: MultiCloudWorkspaceConfig,
                                  ctx: RawlsRequestContext
   ): Future[CreationStatus] = {
-    val projectName = createProjectRequest.projectName.value
+    val projectName = createProjectRequest.projectName
+    var profileModelId = None: Option[UUID]
+    var landingZoneId = None: Option[UUID]
     try {
```

Comment:
Try to avoid java-style `try catch finally` blocks because it encourages imperative-style variable setting and complicates error handling logic (like below) - using error handling with lexical scoping gives better granularity and clarity over the state of the world. See my comment below.